commands/tiering.py

```diff
@@ -409,6 +409,31 @@ async def vsliveguide(self,
             title='New Tiering Members Guide',
             content=f'New Tiering Members Tips and Tricks guide posted in {dest_channel.mention}'),
             ephemeral=True)
+        
+    def verifyRoom(self, room):
```

Comment:
```suggestion
    def verify_room(self, room):
```

Reply:
All camelcase has been swapped to underscores